File: string.js/reverse.py
num=int(input('enter the number'))
a=[9,2,3,4,5,6,7,8,9]
b=[]
i=0
while i<len(a)-num:
    b.append(a[i])
    i=i+1
j=0
while j<=num:
    b.append(a[-j])
    j=j+1
# i=i+1
print(b)

File: string.js/reverse.js
const a=require("readline-sync")
const number=a.questionInt("enter the number")
let array=[1,2,3,4, 5,6,7,3,4,6,8]
empty=[]
for(var i=0;i<array.length-number;i++){
    empty.push(array[i])
    for(var j=0;j<=number;j++){
        empty.push(array[-j])
    }

}
console.log(empty);